clients/cli/src/test/java/org/apache/gravitino/cli/integration/test/TableFormatOutputIT.java

```diff
@@ -145,21 +146,19 @@ public void testMetalakeDetailsCommand() {
   }
 
   @Test
-  public void testCatalogDetailsCommand() {
+  public void testCatalogListCommand() {
```

Comment:
I'm not sure why this has changed from a Details command to a List command don't we want to test both?

Reply:
Sorry, I lost the `testCatalogDetailsCommand()` during the rebase, but I have already reverted it.